account_authentications.go

```diff
@@ -58,6 +118,12 @@ type (
 		APIResponse
 		AuthenticationMethod AccountAuthenticationMethod `json:"authentication_method"`
 	}
+
+	// AccountAuthenticationMethodUpdateResponse AccountAuthenticationMethodUpdate call response
+	AccountAuthenticationMethodUpdateResponse struct {
```

Comment:
The same here, we can reuse `AccountAuthenticationResponse`

Reply:
Please see my prev comment and check spec https://api.aiven.io/doc/#operation/AccountAuthenticationMethodUpdate

Reply:
If you take a closer look on the response section you will find that `authentication_method_type` is required in the response. 
![image](https://user-images.githubusercontent.com/4929004/194236856-f4b7ab68-396e-40be-b9b2-b2c33e849f79.png)

As I said before, let's use one structure for all responses regardless of the HTTP method to mimic API responses.

Reply:
I have dropped replace method yesterday. And PR has been updated.